release/package.json

```diff
@@ -1177,18 +1177,6 @@
 					"default": 4,
 					"description": "The deep level of directory hierarchy when searching for sln/projects"
 				},
-				"FSharp.monoPath": {
-					"type": "string",
-					"default": "mono",
-					"description": "The path to Mono executable",
-					"scope": "machine-overridable"
-				},
-				"FSharp.fsiFilePath": {
```

Comment:
we don't really support .net fx scripting anymore, do we?

Reply:
I think it's still possible to do full framework scripting. As far as I remember it was related to the fact that there are breaking changes between both FSI implementations so users may want to use full framework FSI

Reply:
added fsiFilePath, useSdkScript, and suggestSdkScripts back, as well as the related code in Environment.fs, LanguageService.fs and Fsi.fs